app/javascript/app/components/sectors-agriculture/countries-context/land-area/land-area-component.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandAreaProvider from 'providers/agriculture-land-area-provider';
+import { SunburstChart, Card, Tag, Icon } from 'cw-components';
+import infoIcon from 'assets/icons/info';
+import Tooltip from './tooltip';
+
+import styles from './land-area-styles.scss';
+
+const cardTheme = {
+  card: styles.card,
+  data: styles.data
+};
+
+const LandArea = ({
+  selectedCountry,
+  selectedYear,
+  chartData,
+  chartColors,
+  chartConfig,
+  firstLevelLegend,
+  handleInfoBtnClick
+}) => (
+  <div className={styles.container}>
+    <Card
```

Comment:
I think you can add guard here to show component only when `chartData` is loaded:
`{chartData && ...`

Reply:
👍